file_catalog/server.py

```diff
@@ -332,78 +322,32 @@ def wrapper(self: Any, *args: Any, **kwargs: Any) -> Any:
     return wrapper
```

Comment:
I've checked out the differences, and I think this block can be straight deleted and we can use the
```python
@handler.authenticated
```
decorator.  The main thing the current version has is a pass-through when auth is disabled.  I'm not sure if that would break a test, but production would be fine.

Reply:
`@handler.authenticated` is much simpler. Is this because `self.current_user`/`self.get_current_user()` is doing the actual auth?

Reply:
Yes, that's where much of the code went.